codefresh/destroy.yaml

```diff
@@ -5,6 +5,10 @@ stages:
 
 steps:
   main_clone:
```

Comment:
```suggestion
  ask_for_permission:
```

Reply:
First step in a codefresh pipeline should be called `main_clone`, otherwise codefresh adds it's own `main_clone` step